### ISSUE TITLE
Rails 8 support

### ISSUE THREAD

**Issue by andynu** [smartinez87/exception_notification#545]
### Steps to reproduce
Make a rails project with:
```
gem 'rails', '~> 8.0.0'
gem 'exception_notification', '~> 4.5.0'
```
Attempt to bundle.

### Expected behavior
success.

### Actual behavior
failure:
```
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Could not find compatible versions

Because exception_notification >= 4.5.0 depends on activesupport >= 5.2, < 8
  and rails >= 8.0.0 depends on activesupport = 8.0.0,
  exception_notification >= 4.5.0 is incompatible with rails >= 8.0.0.
So, because Gemfile depends on rails ~> 8.0.0
  and Gemfile depends on exception_notification ~> 4.5.0,
  version solving has failed.
```

### System configuration
**Rails version**: 8.0.0

**Ruby version**: 3.3.5

**Other configurations**:

**Comment by brandondrew** [comment]
The strange thing is that version 4.1.1 claims to support Rails 8, but this is rather doubtful.  It appears more likely that its lack of an upper limit was an oversight.

**Comment by kmcphillips** [comment]
Yeah it downgraded to the last version that didn't have a configured upper limit.

**Comment by Petercopter** [comment]
This is on the way to being resolved 🎉 

https://github.com/smartinez87/exception_notification/pull/544

Thanks @smartinez87

**Comment by brandondrew** [comment]
> This is on the way to being resolved 🎉

@Petercopter 
On the way?  Or already resolved (if we use the master branch)?

**Comment by RomainMorlevat** [comment]
> @Petercopter On the way? Or already resolved (if we use the master branch)?

Using that on Gemfile does the job:
`gem "exception_notification", git: "https://github.com/smartinez87/exception_notification.git", branch: "master"`

**Comment by SebastianPoell** [comment]
Yeah, a new tag would be awesome (last was Jan 2022) :)

**Comment by markets** [comment]
@smartinez87 Sorry for the ping 🙏🏼  but could you please release a new version? It will be really useful for people upgrading to Rails v8. I know we can pin the gem to GitHub master branch, but a new release in RubyGems will make the upgrade more simple for everybody.

Thanks!

**Comment by kmcphillips** [comment]
@smartinez87 @FLarra @mschnitzer All of you have published this gem before. Would it be possible to please do a release. It blocks Rails upgrades and hasn't had a release in years. Just push what's on `main`.

**Comment by akaspick** [comment]
@kmcphillips  Just do what @RomainMorlevat mentioned.... point to the main github branch in your Gemfile to get the current updates.

My Gemfile has:
`gem 'exception_notification', github: 'smartinez87/exception_notification'`

**Comment by kmcphillips** [comment]
I understand that, but [everyone who uses this gem](https://rubygems.org/gems/exception_notification/) shouldn't have to figure out how to do that, just because the maintainers won't cut a release. This last version has been downloaded 4 million times.

**Comment by akaspick** [comment]
Most people are likely still not on Rails 8 and the gem is working fine for them still.

In a perfect world, a new version of the gem would have been released, but it's an open source project, so anyone who can't install the gem with rail 8 will just end up here eventually with the above solution in the mean time.

At least the main branch has the fix. A lot of time, I've had to refer to a completely different fork or fork my own branch of gems to get compatibility working.

**Comment by SebastianPoell** [comment]
Note, that simply adding `github` to that line will not lead to a stable solution - you are just pulling whatever is currently on main/master - which might change any day. You might end up with different version on different machines. Add an additional `ref` to that line in your Gemfile to pin one particular commit.

**Comment by akaspick** [comment]
@SebastianPoell Correct, so `gem 'exception_notification', github: 'smartinez87/exception_notification', ref: '26441fb'` is a better solution.

**Comment by kmcphillips** [comment]
Alright I got collaborator access and rubygems push access.

Anyone who wants to have a look at this PR, it gets the build green and fixes enough I think to push a new version. Feedback welcome:
* https://github.com/smartinez87/exception_notification/pull/551
